src/app/components/AllowanceMenu/index.tsx

```diff
@@ -138,6 +142,20 @@ function AllowanceMenu({ allowance, onEdit, onDelete }: Props) {
                 onChange={(e) => setBudget(e.target.value)}
               />
             </div>
+            <div className="flex items-center mt-6">
+              <Checkbox
+                id="enable_login"
+                name="enable_login"
+                checked={login}
+                onChange={(e) => setLogin(!login)}
+              />
+              <label
```

Comment:
This is the first time we use a checkbox. For everything else we use `Toggle`. Should we use toggle here as well?

Reply:
We also use it in Remember and Set Budget right?

Reply:
But yeah, I think it's not being highlighted much right now, may be we can use toggle and also add hint below it (like we do in settings) to let people know what it is about

Reply:
+1